synapse/lib/storm.py

```diff
@@ -943,6 +920,7 @@ def dmonWarn(evnt):
                 raise
 
             except Exception as e:
+                self._runLogAdd(('err', s_common.excinfo(e)))
```

Comment:
this changes the shape of the items in the runlog. do we want them to all be timestamp, str or timestamp, (random thing) ?

Reply:
They are (tick, event) tuples, so i dont think this changes their shape